### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by area55git**

### CODE REVIEW

**Comment by MalloZup** [comment]
@area55git thx look really nice, one question can we have the image on the dir `help`` inside github?

**Comment by area55git** [comment]
Yes, I do it in a couple of hours

**Comment by MalloZup** [comment]
@area55git  ok look really awesome , i can't wait to put it on readme :cupid:

**Comment by MalloZup** [comment]
link

**Comment by area55git** [comment]
Should I close this PR and open a new one? I do not know much about github

**Comment by MalloZup** [comment]
@area55git  if you are more confortable doing a new PR close this one and open a new one.

As you wish :+1:

**Comment by area55git** [comment]
Chek Please

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# kubeojo
-
-# [What is kubeojo in 10 seconds](/help/README.md)
+<p align="center"><img src="https://s33.postimg.cc/b106fo2gf/LOGO_KUBEOJO_Readme.jpg"></p>
```
Comment (MalloZup) [inline README.md]: looks good until this link is from https

**Comment by MalloZup** [comment]
@area55git  i can merge it is as it is now, and i will add the link on readme.md manually 
or 
we need to adapt the readme.md modifying the right links

as you wish, for me both is fine

File: README.md (hunk 1 above)
Comment (MalloZup) [reply README.md]: we should do link from `help` dir

**Comment by area55git** [comment]
As you want for me it's fine

**Comment by MalloZup** [comment]
@area55git :tada: :tada:  i will modify the links with commit thx

**Comment by area55git** [comment]
@MalloZup We are here to serve you, remember that you can request my collaboration whenever you wish.

**Comment by MalloZup** [comment]
@area55git thank you !!! was really cool to work with you! i wish you a great day and i will call you for collaboration once i have a project ! at moment i need to code the existing one ... :+1: :smile:

**Pull request merged**